src/toniarts/openkeeper/view/PlayerCameraState.java

```diff
@@ -212,6 +212,7 @@ public void doTransition(int sweepFileId, final ActionPoint point) {
             @Override
             public void onPlay(CinematicEvent cinematic) {
                 stateManager.getState(PlayerState.class).setTransitionEnd(false);
+                stateManager.getState(PlayerState.class).setWideScreen(true);

```

Comment:
we have differently trigger with widescreen action. Not shure that we need add this action to transition